src/llms/openai.ts

```diff
@@ -313,7 +313,7 @@ export function streamChatCompletions(request: ChatCompletionsRequest): Observab
   const channel: LiveChannelAddress = {
     scope: LiveChannelScope.Plugin,
     namespace: LLM_PLUGIN_ID,
-    path: OPENAI_CHAT_COMPLETIONS_PATH,
+    path: OPENAI_CHAT_COMPLETIONS_PATH + self.crypto.randomUUID(),
```

Comment:
Just to keep the path a bit neater:

```suggestion
    path: OPENAI_CHAT_COMPLETIONS_PATH + '/' + self.crypto.randomUUID(),
```